test/fuse/linux/fuse_base.h

```diff
@@ -193,6 +193,16 @@ class FuseTest : public ::testing::Test {
     iov[1].iov_base = &payload;                             \
   } while (0)
 
+#define SET_IOVEC_WITH_HEADER_PAYLOAD_NAME(iov, header, payload, name) \
```

Comment:
These can easily be functions. Strongly prefer functions over macros whenever possible: https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros